### PR TITLE
make-ca: update to 1.9.

### PR DESCRIPTION
**Pull request by knusbaum**
#### Testing the changes
- I tested the changes in this PR:  **briefly**

### CODE REVIEW

**Comment by ericonr** [comment]
Please mention in the commit that you're switching upstream to the version that has the newest releases.

File: srcpkgs/make-ca/template [hunk 1]
```diff
@@ -1,7 +1,7 @@
 # Template file for 'make-ca'
 pkgname=make-ca
-version=1.7
-revision=2
+version=1.9
```
Comment (paper42) [inline srcpkgs/make-ca/template:3]: 1.10 is out

File: srcpkgs/make-ca/template [hunk 2]
```diff
@@ -1,7 +1,7 @@
...
+revision=1
 wrksrc="make-ca-${version}"
```
Comment (paper42) [inline srcpkgs/make-ca/template:5]: this line can be removed, $wrksrc is $pkgname-$version by default

File: srcpkgs/make-ca/template [hunk 3]
```diff
@@ -10,6 +10,6 @@ depends="p11-kit"
 short_desc="PKI setup script for LFS (and others)"
 maintainer="Kyle Nusbaum <knusbaum+void@sdf.org>"
 license="GPL-3.0-or-later"
-homepage="https://github.com/djlucas/make-ca/"
-distfiles="https://github.com/djlucas/make-ca/releases/download/v${version}/make-ca-${version}.tar.xz"
-checksum=6db8b96c961790507c5e36e0ed75a079ae95300f520cd88ac061cf44a4733c2f
+homepage="https://github.com/lfs-book/make-ca/"
```
Comment (paper42) [inline srcpkgs/make-ca/template:13]:
```suggestion
homepage="https://github.com/lfs-book/make-ca/"
changelog="https://raw.githubusercontent.com/lfs-book/make-ca/master/CHANGELOG"
```

**Pull request closed without merging**